### ISSUE TITLE
Arms Warrior Sim issue

### ISSUE THREAD

**Issue by Darthius** [simulationcraft/simc#3869]
For some reason when I sim my Arms spec I'm only getting a DPS output of 500k, although I'm pulling 900+?

It works fine for the Fury spec but on Arms it's simming far to low with the same settings.

**Comment by vituscze** [comment]
Upload the profile you used for simming Arms.

**Comment by Darthius** [comment]
```
# Profile will be downloaded into a new tab.
#
# Clicking Simulate will create a simc_gui.simc profile for review.
warrior="Darthioor"
level=110
race=orc
region=eu
server=stormscale
role=attack
professions=blacksmithing=800/mining=800
talents=2312222
spec=arms
artifact=36:0:0:0:0:1136:1:1137:1:1138:1:1139:1:1140:1:1141:1:1142:1:1143:4:1144:3:1145:4:1146:4:1147:4:1148:4:1149:4:1150:4:1151:3:1356:1:1393:1:1613:4:1614:1:1615:1:1714:1

head=,id=137410,bonus_id=3418/1542/3528
neck=,id=132444,enchant_id=5439,bonus_id=1811/3455,gem_id=130246
shoulder=,id=147070,enchant_id=5883,bonus_id=3562/1497/3528
back=,id=138374,enchant_id=5434,bonus_id=3516/1487/3528
chest=,id=140913,bonus_id=3517/1808/1492/3528,gem_id=130220
tabard=,id=35221
wrist=,id=140902,bonus_id=3444/1492/3336
hands=,id=138354,enchant_id=5445,bonus_id=3516/1808/43/1487/3528,gem_id=130220
waist=,id=144497,bonus_id=3417/1552/3337
legs=,id=134506,bonus_id=3573/1552/3337
feet=,id=140891,bonus_id=3516/1502/3337
finger1=,id=140897,enchant_id=5428,bonus_id=3517/1497/3336
finger2=,id=137052,enchant_id=5428,bonus_id=3529/3530/1811,gem_id=130220
trinket1=,id=140806,bonus_id=3514/1472/3528
trinket2=,id=140799,bonus_id=3516/1497/3336
main_hand=,id=128910,bonus_id=750,relic_id=3516:1492:3336/3517:1492:3528/3561:1487:3336,gem_id=140816/140820/147109/0
```

**Comment by Darthius** [comment]
Excuse the text not needed....

**Comment by vituscze** [comment]
Running the above profile, I get
```
 865120  100.0%  Darthioor
```

**Comment by Darthius** [comment]
Very strange, have downloaded the most recent version and again its at 357k, works fine on my alts just not Arms Warrior

**Comment by vituscze** [comment]
Can you upload `simc_gui.simc` file? Run the sim that's giving you trouble, you can then find that file in the directory where your simc is installed.

**Comment by Darthius** [comment]
Found the file but it won't let me paste it into this, says "We don't support that file type".

**Comment by Darthius** [comment]
Pasting the file contents of the "simc_gui" file in my Simcraft folder, hope this is what you need ?- 

```
### Begin GUI options ###
ptr=0
item_db_source=ptrhead/local/bcpapi/wowhead
target_error=0
iterations=10000
default_world_lag=0.1
max_time=300
vary_combat_length=0.2
fight_style=Patchwerk
tmi_window_global=6
target_level+=3
target_race=Humanoid
default_skill=1.0
optimal_raid=0
override.bloodlust=1
override.bleeding=1
override.mortal_wounds=1
threads=4
process_priority=Low
calculate_scale_factors=1
scale_lag=1
scale_only=str,agi,sta,int,spi,sp,ap,crit,haste,mastery,vers,wdps,wohdps,armor,bonusarmor,leech,runspeed
statistics_level=1
### End GUI options ###
### Begin simulateText ###
# Profile will be downloaded into a new tab.
#
# Clicking Simulate will create a simc_gui.simc profile for review.
warrior="Darthioor"
level=110
race=orc
region=eu
server=stormscale
role=attack
professions=blacksmithing=800/mining=800
talents=2312222
spec=arms
artifact=36:0:0:0:0:1136:1:1137:1:1138:1:1139:1:1140:1:1141:1:1142:1:1143:4:1144:3:1145:4:1146:4:1147:4:1148:4:1149:4:1150:4:1151:3:1356:1:1393:1:1613:4:1614:1:1615:1:1714:1

head=,id=137410,bonus_id=3418/1542/3528
neck=,id=132444,enchant_id=5439,bonus_id=1811/3455,gem_id=130246
shoulder=,id=147070,enchant_id=5883,bonus_id=3562/1497/3528
back=,id=138374,enchant_id=5434,bonus_id=3516/1487/3528
chest=,id=140913,bonus_id=3517/1808/1492/3528,gem_id=130220
tabard=,id=35221
wrist=,id=140902,bonus_id=3444/1492/3336
hands=,id=138354,enchant_id=5445,bonus_id=3516/1808/43/1487/3528,gem_id=130220
waist=,id=144497,bonus_id=3417/1552/3337
legs=,id=134506,bonus_id=3573/1552/3337
feet=,id=140891,bonus_id=3516/1502/3337
finger1=,id=140897,enchant_id=5428,bonus_id=3517/1497/3336
finger2=,id=137052,enchant_id=5428,bonus_id=3529/3530/1811,gem_id=130220
trinket1=,id=140806,bonus_id=3514/1472/3528
trinket2=,id=140799,bonus_id=3516/1497/3336
main_hand=,id=128910,bonus_id=750,relic_id=3516:1492:3336/3517:1492:3528/3561:1487:3336,gem_id=140816/140820/147109/0

### Gear from Bags
#
# Ceann-Ar Charger (940)
# head=,id=137088,bonus_id=1811/3530
#
# Warped-Plate Helmet (369)
# head=,id=121363
#
# Warhelm of the Obsidian Aspect (890)
# head=,id=138357,bonus_id=3514/1487/3337
#
# Cruel Gladiator's Plate Helm (895)
# head=,id=144771,bonus_id=3568/1680/1527/3337
#
# Casque of the Deep (900)
# head=,id=134512,bonus_id=3417/1808/1552/3337
#
# Ferocious Gladiator's Pendant (880)
# neck=,id=145499,bonus_id=3567/41/1512/3528
#
# Cruel Gladiator's Choker (895)
# neck=,id=144659,bonus_id=3568/1808/1527/3337
#
# Animus Pendant of the Harmonious (895)
# neck=,id=147263,bonus_id=1782/3574/3144/3337
#
# Brooch of the Astral Scryer (880)
# neck=,id=140900,bonus_id=3514/1477/3336
#
# Shoulderplates of Crackling Flame (900)
# shoulder=,id=147069,bonus_id=3561/42/1482/3528
#
# Pauldrons of Encroaching Limits (900)
# shoulder=,id=142128,bonus_id=3417/1512/3337
#
# Frozen Pendulum Shoulderguards (890)
# shoulder=,id=134518,bonus_id=3418/1542/3528
#
# Shoulderplates of the Obsidian Aspect (895)
# shoulder=,id=138363,enchant_id=5883,bonus_id=3516/1492/3336
#
# Ferocious Gladiator's Plate Shoulders (880)
# shoulder=,id=145617,bonus_id=3567/1721/1512/3528
#
# Cloak of Growing Mistrust (905)
# back=,id=147764,enchant_id=5434,bonus_id=3572/1487/3336
#
# Titanic Onslaught Breastplate (900)
# chest=,id=147187,bonus_id=3561/41/1482/3528
#
# Arcane Defender's Breastplate (890)
# chest=,id=134267,bonus_id=3418/1552/3528
#
# Chestplate of the Obsidian Aspect (875)
# chest=,id=138351,bonus_id=3514/1472/1813
#
# Orgrimmar Tabard (100)
# tabard=,id=118372
#
# Gauntlets of Dreadful Tenacity (900)
# hands=,id=147748,bonus_id=3572/1482/3528
#
# Gauntlets of Fractured Eons (895)
# hands=,id=140879,enchant_id=5445,bonus_id=3444/1492/3336
#
# Arcane Defender's Belt (890)
# waist=,id=134269,bonus_id=3418/1552/3528
#
# Reaver's Rattling Girdle (905)
# waist=,id=147753,bonus_id=3572/1487/3336
#
# Cruel Gladiator's Plate Girdle (870)
# waist=,id=144777,bonus_id=3568/1711/1502/3528
#
# Rethu's Incessant Courage (970)
# waist=,id=146667,bonus_id=1811/3570,gem_id=130220/130220/130220
#
# Legplates of the Obsidian Aspect (875)
# legs=,id=138360,bonus_id=3514/1472/1813
#
# Ferocious Gladiator's Plate Legguards (880)
# legs=,id=145615,bonus_id=3567/1715/1512/3528
#
# Aggramar's Stride (940)
# feet=,id=132443,bonus_id=3459/3530
#
# Ring of Looming Menace (880)
# finger1=,id=134533,bonus_id=3573/1532/3528
#
# Scaled Band of Servitude (900)
# finger1=,id=147020,enchant_id=5428,bonus_id=3564/1482/3337
#
# Sephuz's Secret (940)
# finger1=,id=132452,enchant_id=5428,bonus_id=3529/3530/1811,gem_id=130220
#
# Mindrend Band (880)
# finger1=,id=138220,enchant_id=5429,bonus_id=3573/1808/1502/3528,gem_id=130220
#
# Bloodstained Handkerchief (885)
# trinket1=,id=142159,bonus_id=3573/1502/3336
#
# Draught of Souls (880)
# trinket1=,id=140808,bonus_id=3515/1808/1477/1813,gem_id=130220
#
# Cradle of Anguish (900)
# trinket1=,id=147010,bonus_id=3561/1482/3528
#
# Peon's Mining Pick (578)
# main_hand=,id=116913
#

### End simulateText ###
desired_targets=1
### Begin overrides ###
# User-specified persistent global and player parameters will be set here.

### End overrides ###
### Begin final options ###
### End final options ###
### END ###
```

**Comment by vituscze** [comment]
Cannot reproduce, getting correct numbers on my end.

**Comment by vituscze** [comment]
Which version of simc are you using?

**Comment by wr8th3** [comment]
I had this same issue, however, I realized my mistake was downloading an older version (720-02). I de-installed and re-installed the current version and numbers went back to normal.

I got that version number in my head from the front page advertisement that a new version was available, but it's advertising 720-02, so when I went to the nightly build, I blindly followed the version listed and grabbed it.
